threshsign/src/eddsa/EdDSAMultisigVerifier.cpp

```diff
@@ -119,8 +119,8 @@ bool EdDSAMultisigVerifier::verify(const char *msg, int msgLen, const char *sig,
 int EdDSAMultisigVerifier::requiredLengthForSignedData() const {
   return static_cast<int>(static_cast<unsigned long>(signersCount_) * sizeof(SingleEdDSASignature));
 }
-const IPublicKey &EdDSAMultisigVerifier::getPublicKey() const { return verifiers_[0].getPubKey(); }
+const IPublicKey &EdDSAMultisigVerifier::getPublicKey() const { return verifiers_[0].getPublicKey(); }
```

Comment:
same comment as in the signer class

Reply:
`const PublicKeyType &getPublicKey() const;` removed.
`const PublicKeyType publicKey_;` changed from `private` to `public`.
Changes in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2